### ISSUE TITLE
I couldn't find any linked video skills

### ISSUE THREAD

**Issue by rotaru01** [dmhacker/alexa-youtube-skill#36]
Hey, Please help me with this error. 
I receive this error when i try to play youtube music. I followed exactly your steps and all was setup succesfully. 
Now when i am trying to play a melody it says: I couldn't find any linked video skills that can find that. 
Can you help me please? Thanks. Also what is the command to play a melody?

**Comment by dmhacker** [comment]
Did you enable your skill for testing?

**Comment by sopeti** [comment]
Hello dmhacker, great work with this .. 

I have followed the steps and have also cross-checked 2-3 times for any mistakes on my side, but all is fine.I am getting the same error as stated above when running test from ASK Developer console:

  "code": "SKILL_ENDPOINT_ERROR",
  "description": "An error occurred while issuing a SpeechletRequest for .."
  "type": "INVALID_RESPONSE",
   "message": "An exception occurred while dispatching the request to the skill."

                        "locale": "en-US",
                            "intent": {
                                "name": "GetVideoIntent",
                                "confirmationStatus": "NONE",
                                "slots": {
                                    "VideoQuery": {
                                        "name": "VideoQuery",
                                        "value":  "songs",
                                        "resolutions": {
                                            "resolutionsPerAuthority": [
                                                {
                                                    "authority": "amzn1.er-authority.echo-sdk.amzn1.ask.skill.25af997f-2fb9-45b3-bd81-30354fa15e1c.VIDEOS",
                                                    "status": {
                                                        "code": "ER_SUCCESS_NO_MATCH"
                                                    }
                                                }
                                            ]
                                        },
                                        "confirmationStatus": "NONE"
    

could the location of the api endpoint being in EU be a problem? 
"apiEndpoint": "https://api.eu.amazonalexa.com",
my lambda function is deployed in Far-east ...  dont think it should matter though..

Please help !

**Comment by c-kobo** [comment]
Hi,
same issue here...I also followed the steps precisely as described and get the "couldn't find any linked video skills...' error. I have set up my API Endpoint to Ireland for German users. 

Any help is appreciated!

**Comment by dmhacker** [comment]
@sopeti @c-kobo 

There could be a variety of issues causing this. Have you guys made sure to use the correct ARN? Additionally, make sure that the ZIP file you've uploaded to the Lambda server isn't nesting anything. If you open it up, you should be greeted with a js/ folder and an index.js file.

**Comment by sopeti** [comment]
@dmhacker  Thank you so much for paying attention.  Yes, the ARN i have used is correct. 
Also, I have ensured that the folder structure on the Lambda server is same as your setup depicted in your guide at https://imgur.com/a/H5R7L

**Comment by sopeti** [comment]
@dmhacker , update:  this is now working as expected, after adding the trigger "Amazon Skill Set"  to the lambda ! thanks, again !!

**Comment by dmhacker** [comment]
@sopeti 

Glad to hear that.

@c-kobo 

How is it working out for you?

**Comment by c-kobo** [comment]
@dmhacker Still no success. Arn and folder structure looks good. 
@sopeti did you mean Alexa skill set? I couldn’t find any Amazon skill set.
I will check with your guide step by step, maybe I missed out something.

**Comment by sopeti** [comment]
@c-kobo  yes,  I went to the AWS Lambda, and added "Amazon Skill Set" as a trigger.  
You should see this option on top of the page, once you are on the Lambda page.

Also, don't forget to configure AWS Lambda to set ALEXA_APPLICATION_ID as given in @dmhacker's detailed guide.

Good luck!

**Comment by c-kobo** [comment]
Hi all, just returned from vacation and went through it again. It is working now, the issue was the way I was asking, I always tried with 'Play <song xyz> on youtube', which was failing. I got it now running by utterances like this: 'Ask youtube for <song xyz>'. Works great now, thanks dmhacker.

**Comment by jbritz** [comment]
I am still getting the same problem on the Echo dot.
I am probably just missing something simple.
Any help would be appreciated.

**Comment by geek-coder** [comment]
Use Google home... Instead of Alexa. :-)
Amazon won't allow this as skill
